src/main/java/com/skhu/luxuryshop/product/controller/ProductController.java

```diff
@@ -0,0 +1,48 @@
+package com.skhu.luxuryshop.product.controller;
+
+import com.skhu.luxuryshop.product.dto.ProductRequestDto;
+import com.skhu.luxuryshop.product.service.ProductService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/products")
+@Slf4j
+public class ProductController {
+    private final ProductService productService;
+
+    @GetMapping
+    public ResponseEntity getProducts() {
+        return ResponseEntity.ok(productService.findAll());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity getProductById(@PathVariable Long id) {
+        return ResponseEntity.ok(productService.findById(id));
+    }
+
+    @PostMapping
+    public ResponseEntity<Long> registerProduct(@RequestBody @Valid ProductRequestDto productDto) {
+        productService.save(productDto);
+        return new ResponseEntity("상품등록 성공", HttpStatus.OK);
+    }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<Long> updateProduct(@PathVariable Long id, @RequestBody @Valid ProductRequestDto productRequestDto) {
+        productService.update(id, productRequestDto);
+        return new ResponseEntity("상품업데이트 성공", HttpStatus.OK);
+    }
```

Comment:
왜냐면 HttpStatus.OK를 보내주고 있기때문에 상품 업데이트 성공했다는 메세지는 응답할 필요가 없어보입니다. :)
업데이트시 필요한 데이터가 있다면 그걸 보내주면 되고 없다면 아무것도 안보내줘도 됩니다!
```suggestion
    @PutMapping("/{id}")
    public ResponseEntity<Long> updateProduct(@PathVariable Long id, @RequestBody @Valid ProductRequestDto productRequestDto) {
        productService.update(id, productRequestDto);
        return ResponseEntity.ok().build();
    }
```